db/migrate/20180802145655_add_address_to_schools.rb

```diff
@@ -0,0 +1,12 @@
+class AddAddressToSchools < ActiveRecord::Migration
+  def change
+    add_column :schools, :address_line_1, :string
+    add_column :schools, :address_line_2, :string
+    add_column :schools, :state, :string
+    add_column :schools, :postal_code, :string
+    add_column :schools, :phone_number, :string
+    add_column :schools, :borough, :string
+    add_column :schools, :created_at, :datetime
+    add_column :schools, :updated_at, :datetime
+  end
+end
```

Comment:
One more change:  currently, the z code is stored in the "code" column of the schools table, as a plain string.  We're planning to have a zcode lookup table, with the columns:  id (MLN primary key), sierra_id (Sierra primary key), code (the string zcode).  And then the code column of the schools table would become a foreign key.

This would allow us to send the correct numeric pcode4 id to Sierra API (the pcode4 corresponds to the Sierra id primary key, rather than to the zcode string).